content/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry.md

```diff
@@ -163,6 +163,24 @@ For more information on creating a package, see the [maven.apache.org documentat
 
 {% data reusables.package_registry.viewing-packages %}
 
+### Using a package
+
+To use an Apache Maven package from {% data variables.product.prodname_registry %}, edit the *pom.xml* file to include the package as a dependency. If you want to use packages from more than one repository, add a `repository` tag for each. For more information on using a *pom.xml* file in your project, see "[Introduction to the POM](https://maven.apache.org/guides/introduction/introduction-to-the-pom.html)" in the Apache Maven documentation.
```

Comment:
Should we clarify the difference between using vs installing?

```suggestion
To use an Apache Maven package from {% data variables.product.prodname_registry %} without installing the package, edit the *pom.xml* file to include the package as a dependency. If you want to use packages from more than one repository, add a `repository` tag for each. For more information on using a *pom.xml* file in your project, see "[Introduction to the POM](https://maven.apache.org/guides/introduction/introduction-to-the-pom.html)" in the Apache Maven documentation.
```

Reply:
Thanks @skedwards88 ! I'm going to add a clarification about it :raised_hands:

Reply:
Sorry @skedwards88 , when reading the **Installation** section again, I realized that it could be wrong. Now I understand that I felt confused when reading it at the first time.

Please, maybe I'm wrong :pray:  I'm going to explain how I usually use Maven or Gradle.

As far as I know, the dependencies are used to create another artifact and `install` will install that new artifact into the local repository (`~/.m2/repository`). 

I mean, `install` won't install the included dependencies like:

```
dependencies {
    implementation 'com.example:package'
}
```

But `install` will install the artifact that needs `com.example:package` as a dependency to be built. And it will be installed into the local repository (`~/.m2/repository`).

So I think it would be just necessary to explain:

* How to publish a package into the GitHub Packages repository **(the existing section)**
* How to use that published package as a dependency to create new packages **(the proposed new section)**

I'm not sure if the **Installation** section makes sense and it's right. However, I'm talking from my humble experience. Maybe the author thought about a different use case that I don't know.

Looking forward to knowing your feedback about it! Please, let me know if I didn't explain myself.

Thank you so much for your patience :pray: and thanks again for your review!